core/templates/pages/skill-editor-page/skill-preview-tab/skill-preview-tab.component.ts

```diff
@@ -145,6 +148,20 @@ export class SkillPreviewTabComponent implements OnInit, OnDestroy {
       });
   }
 
+  isSupplementalCardNonempty(card: StateCard): boolean {
+    return !card.isInteractionInline();
+  }
+
+  canWindowShowTwoCards(): boolean {
+    return this.windowDimensionsService.getWidth() >
+    ExplorationPlayerConstants.TWO_CARD_THRESHOLD_PX;
+  }
+
+  isCurrentSupplementalCardNonempty(): boolean {
```

Comment:
One nit, can you capitalize 'empty' to make the name `isSupplementalCardNonEmpty`

Reply:
Thanks! @Lawful2002 Made the suggested changes :)